src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -219,12 +220,11 @@ public Size Size
         }
 
         /// <inheritdoc />
-        public double FramesPerSecond {
-            get => framesPerSecond;
+        public double FramesPerSecond
+        {
+            get => Math.Round(1.0 / renderPeriod);
```

Comment:
Actually on second thought, why are we rounding? What if the user explicitly wants a `29.95` FPS window? It should be up to the user to round it should they need to ;)